server/rosetta/go.mod

```diff
@@ -0,0 +1,114 @@
+module github.com/cosmos/cosmos-sdk/rosetta
```

Comment:
Should we have `cosmossdk.io/rosetta` instead ? @tac0turtle

Reply:
yea lets do it